platform/linuxbsd/wayland/wayland_thread.cpp

```diff
@@ -101,7 +101,7 @@ Vector<uint8_t> WaylandThread::_read_fd(int fd) {
 		data.resize(bytes_read + chunk_size);
 	}
 
-	return data;
+	return Vector<uint8_t>(data);
```

Comment:
CC @Riteo to confirm that have `data` as `LocalVector` to convert on return is worth it here, as opposed to making it `Vector` from the start (I see it's resized quite a bit so I suppose the current approach is a good option).